<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordDataGen</class>
 <widget class="QDialog" name="WordDataGen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>17</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sentence Generation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/share/machinetrans/resources/logo.png</normaloff>../../../../usr/share/machinetrans/resources/logo.png</iconset>
  </property>
  <widget class="QPushButton" name="acceptBttn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Random Sentence Generation</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="quitBttn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>520</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>780</width>
     <height>441</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backBttn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>520</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="helpBttn">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>helpBttn</tabstop>
  <tabstop>listDisplay</tabstop>
  <tabstop>quitBttn</tabstop>
  <tabstop>backBttn</tabstop>
  <tabstop>acceptBttn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
